src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -33,9 +33,6 @@
     <PublishReadyToRun>false</PublishReadyToRun>
     <PublishReadyToRunShowWarnings>false</PublishReadyToRunShowWarnings>
   </PropertyGroup>
-  <ItemGroup>
-    <None Remove="StaticResources\templatesv2.json" />
-  </ItemGroup>
```

Comment:
Just for my own learning, why do we need to remove this?

Reply:
It was already added in the next item group (just below this code) with correct details therefore this config is not being used.